docker-compose.yml

```diff
@@ -67,6 +69,29 @@ services:
     networks:
       - psp
 
+  ####################### Proxy #######################
+  proxy:
+    profiles:
+      - all
+      - psp
+    restart: on-failure:1
+    container_name: psp-proxy
+    build:
+      context: source/backend
+      dockerfile: Dockerfile.proxy
+      args:
+        BUILD_CONFIGURATION: Debug
+    env_file: source/backend/proxy/.env
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Docker
+    ports:
+      - ${API_HTTPS_PORT:-5003}:443
+      - ${API_HTTP_PORT:-5002}:8080
```

Comment:
I know we currently don't use docker for the API container but these env vars should be different than the one for the API (ie PROXY_HTTPS_PORT) or else we risk creating containers that open the same port on the host which will fail.

This also applies to new microservices that are added to this file (eg: for document service it could be DOC_HTTPS_PORT, etc)